src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -355,21 +355,16 @@ private static (EditorTestWorkspace workspace, IDiagnosticAnalyzerService analyz
         var analyzerReference = new TestAnalyzerReferenceByLanguage(DiagnosticExtensions.GetCompilerDiagnosticAnalyzersMap());
         workspace.TryApplyChanges(workspace.CurrentSolution.WithAnalyzerReferences([analyzerReference]));
 
-        var diagnosticService = workspace.GetService<IDiagnosticAnalyzerService>();
+        var diagnosticService = workspace.Services.GetRequiredService<IDiagnosticAnalyzerService>();
```

Comment:
seems no longer necessary in all the tests with the CodeFixService no longer taking it as a param?